.github/workflows/nightly-build.yaml

```diff
@@ -2,7 +2,7 @@
   name: "Nightly build",
   on: {
     schedule: [
-      { cron: "05 16 * * *" }
+      { cron: "20 17 * * *" }
```

Comment:
Should publish a nightly build at 10:20 PT (17:20 UTC).  Assuming everything works.